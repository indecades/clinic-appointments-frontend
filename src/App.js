//imports
import React from 'react';
import { Route, Routes } from 'react-router-dom'
import './App.css';
import Home from './components/Home';
import AdminDashboard from './components/AdminDashboard';
import AdminLogin from './components/AdminLogin';
import Appointments from './components/Appointments';
import CreateNewAdmin from './components/CreateNewAdmin';

function App() {
  {/* creating routes */}
  return ( 
    <div className="App">
      <Routes>
        <Route path='https://clinic-appointment-backend-2dvo.onrender.com/' element={<Home />}></Route>
        <Route path='https://clinic-appointment-backend-2dvo.onrender.com/appointments' element={<Appointments />}></Route>
        <Route path='https://clinic-appointment-backend-2dvo.onrender.com/admin-login' element={<AdminLogin />}></Route>
        <Route path='https://clinic-appointment-backend-2dvo.onrender.com/admin-dashboard' element={<AdminDashboard />}></Route>
        <Route path='https://clinic-appointment-backend-2dvo.onrender.com/create-new-admin' element={<CreateNewAdmin />}></Route>
      </Routes>
    </div>
  );
}

export default App;
