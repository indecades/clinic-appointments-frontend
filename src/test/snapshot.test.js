import React from "react";
import renderer from 'react-test-renderer'
import { MemoryRouter } from "react-router-dom";
import App from "../App";

// snapshot test created of App.js
it('Snapshot of App.js', ()=> {
    const tree = renderer.create(
        <MemoryRouter>
            <App/>
        </MemoryRouter>
    ).toJSON()
    expect(tree).toMatchSnapshot()
}) 
