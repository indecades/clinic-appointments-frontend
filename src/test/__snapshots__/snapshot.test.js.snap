// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of App.js 1`] = `
<div
  className="App"
/>
`;
